utils/PremiumTypeUtils.tsx

```diff
@@ -7,13 +7,6 @@ export enum PREMIUM_RANK {
 export const TEST_PREMIUM_DAYS = 3
 
 export const PREMIUM_TYPES: PremiumType[] = [
-    {
-        productId: 'premium',
-        label: 'Premium',
-        durationString: 'month',
-        priority: PREMIUM_RANK.PREMIUM,
-        options: generateNumberOptionArray(1, 12, 'premium', 1800)
-    },
```

Comment:
This might break the current transaction history logic in TransactionHistory.tsx:
If the transaction productId is "premium_plus....", it first matches with premium_plus and then with premium. Until now that wasn't a problem as premium was the first to be compared

```
 let purchasedPremiumOption
        PREMIUM_TYPES.forEach(premiumType => {
            if (transaction.productId.startsWith(premiumType.productId)) {
                purchasedPremiumOption = premiumType.label
            }
        })
```

Reply:
Ah, I see, good catch. I reverted my changes.